manifests/overlays/additional/jupyter-amd-pytorch-notebook-imagestream.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/opendatahub-io/notebooks/blob/main/jupyter/amd/pytorch"
+    opendatahub.io/notebook-image-name: "AMD-PyTorch"
+    opendatahub.io/notebook-image-desc: "Jupyter AMD optimized PyTorch notebook image for ODH notebooks."
+    opendatahub.io/notebook-image-order: "14"
+  name: jupyter-amd-pytorch
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    # N Version of the image
+    - annotations:
+        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"},{"name":"AMD-PyTorch","version":"2.3"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"AMD-PyTorch","version":"2.3"},{"name":"Tensorboard","version":"2.16"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.8"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"2.2"},{"name":"Scikit-learn","version":"1.4"},{"name":"Scipy","version":"1.12"},{"name":"Elyra","version":"3.16"},{"name":"PyMongo","version":"4.6"},{"name":"Pyodbc","version":"5.1"}, {"name":"Codeflare-SDK","version":"0.15"}, {"name":"Sklearn-onnx","version":"1.16"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.3"}]'
+        openshift.io/imported-from: quay.io/opendatahub/workbench-images
+        opendatahub.io/workbench-image-recommended: 'true'
+        opendatahub.io/notebook-build-commit: 'c010eb0'
+      from:
+        kind: DockerImage
+        name: quay.io/opendatahub/workbench-images@sha256:99e4822a580e8687f178e772af6b3dea507b50b4023792aa2dea0c2475c74ba6
```

Comment:
is this the correct image? when the work on this PR resumes, the image hashes will be changed anyways, so no point worrying about it too much rn

Reply:
seem like a placeholder.